File: Go/Recursions/removeTabs.go
package main

func removeTabs(str string) string {
	if len(str) == 0 {
		return ""
	}

	if  str[0] == '\t' || str[0] == ' ' {
		return removeTabs(str[1:])
	}

	return string(str[0]) + removeTabs(str[1:])
}

File: JavaScript/Recursions/removeTabs.js
const removeTabs = (str) => {
    if(str.length === 0) {
        return "";
    }

    if(str[0] == '\t' || str[0] == ' ') {
        return removeTabs(str.slice(1));
    }
    return str[0] + removeTabs(str.slice(1));
}

console.log(removeTabs("H e l l o W o r l d"));